src/views/Chatting.vue

```diff
@@ -152,6 +151,9 @@ export default {
       localStorage.setItem("uid", user.uid);
     }
 
+    this.roomId = this.$route.params.channelId;
+    localStorage.setItem("room_id", this.roomId);

```

Comment:
@samymokhtari Tu penses que c'est OK de stocker comme ça dans this ET dans le localstorage ? ou nique c'est bon ?

Reply:
Comme c'est un truc qui est amené à changer souvent je verrais mieux ca en sorte de state ou genre variable global un peu comme les redux mais vas'y je vois pas comment faire rapidement on va dire et si ca marche comme ca ca passe c'est carré